app/frontend/src/cljs/teet/file/file_view.cljs

```diff
@@ -304,8 +312,11 @@
                  {:date (format/date-time (:meta/created-at file))
                   :author (user-model/user-name
                             (:meta/creator file))}))]
-        (when status
-          [typography/SmallBoldText (tr-enum status)])]]]
+        [:div {:class [(<class common-styles/flex-align-center)]}
```

Comment:
does this work? I thought that you had to have :classes if you wanted to provide a list of classes to an element?